package.json

```diff
@@ -122,8 +125,10 @@
     "chai-dom": "^1.12.0",
     "compression-webpack-plugin": "^11.1.0",
     "concurrently": "^8.2.2",
+    "cpy-cli": "^5.0.0",
     "cross-env": "^7.0.3",
     "danger": "^11.3.1",
+    "date-fns-v3": "https://registry.npmjs.org/date-fns/-/date-fns-3.6.0.tgz",
```

Comment:
Did you try with?
```suggestion
    "date-fns-v3": "npm:date-fns@3.6.0",
```

Reply:
Nice, thanks for the suggestion. Looks way cleaner. 👍